utils/lshw.go

```diff
@@ -498,6 +508,32 @@ func (l *Lshw) xStorageController(node *LshwNode) *common.StorageController {
 		PhysicalID:               node.Physid,
 		BusInfo:                  node.Businfo,
 	}
+
+	// Parse out PCI Vendor and Product identifiers from node.ProductName if available
+	pciIdRegex, _ := regexp.Compile(` \[(\S+):(\S+)\]$`)
+	if matches := pciIdRegex.FindStringSubmatch(sc.ProductName); matches != nil {
+		sc.ProductName = pciIdRegex.ReplaceAllString(sc.ProductName, "")
+		sc.Common.PCIVendorID = matches[1]
+		sc.Common.PCIProductID = matches[2]
```

Comment:
A len() check here would be ideal, since this could panic if `len(matches) == 1`